Bugzilla/Install/DB.pm

```diff
@@ -829,6 +829,9 @@ sub update_table_definitions {
   $dbh->bz_add_column('oauth2_client', 'hostname',
     {TYPE => 'varchar(255)', NOTNULL => 1, DEFAULT => "''"});
 
+  # Bug 1512484 - kohei.yoshino@gmail.com
```

Comment:
Maybe this should be changed to `dkl@mozilla.com`? Though I don't know what purpose the email address serves here.

Reply:
Yeah in that file it is consistent with other changes that list the bug id and author. And since Kohei wrote that code and the address is still valid, it is best to leave it like that.